site/build.js

```diff
@@ -52,7 +52,16 @@ metalSmith(__dirname)
     .use(function (files, metalsmith, next) {
         var metadata = metalsmith.metadata();
 
-        var assertionsByType = {};
+        // Make sure that the most important types are listed first and in this order:
+        var assertionsByType = {
+            any: [],
+            array: [],
+            boolean: [],
+            function: [],
+            number: [],
+            object: [],
+            string: []
+        };
```

Comment:
Sorry for keep coming back to this sorting issue, but wouldn't sorting by the lowercase type also put `any' on top. The reason I ask, is that I liked that the order was well defined from the users perspective and that you didn't needed to hardcode any names when adding new documentation.